docs/DeveloperGuide.md

```diff
@@ -856,6 +856,48 @@ testers are expected to do more *exploratory* testing.
     1. Other incorrect delete commands to try: `vdelete`, `vdelete x`, `...` (where x is larger than the list size or a non-integer)<br>
        Expected: Similar to previous.
 
+### Deleting an event
+
+1. Deleting an event.
+    1. Prerequisites: List all events using the `elist` command. Multiple events in the list. If the list is empty,
+       create it with the following example,
+       `ecreate n/Food packing r/10 chef r/10 packer sd/23/10/2023 1900 ed/23/10/2023 2200 l/hougang dsc/Packing food for the needy m/100 packets b/100.00 vs/50`
+
+    1. Test case: `edelete 1`
+       Expected: The first event in the displayed event list is deleted successfully. Detail is shown in the status message.
+
+    1. Test case: `edelete`
+       Expected: No change to the event list panel. Invalid command format error is shown in the status message.<br>
+
+    1. Test case: `edele`
+       Expected: No change to the event list panel. Invalid command format error is shown in the status message.<br>
+
+### Editing an event
+1. Editing the detail of an event.
+    1. Prerequisites: List all events using the `elist` command. Multiple events in the list. If the list is empty, 
+       create one with the following example, 
+       `ecreate n/Food packing r/10 chef r/10 packer sd/23/10/2023 1900 ed/23/10/2023 2200 l/hougang dsc/Packing food for the needy m/100 packets b/100.00 vs/50`
+
+    1. Test case: `eedit 1 l/NUS`
+       Expected: The location of the first event in the displayed event list is edited from to NUS successfully. 
+       Detail is shown in the status message.
+
+    1. Test case: `eedit 1 sd/22/10/2023 1900`
+       Expected: The start date of the first event in the displayed event list is edited to 22/10/2023 1900 successfully.
+       Detail is shown in the status message.
```

Comment:
similar test cases, just combine both into one test case `eedit 1 l/NUS sd/22/10/2023 1900`

Reply:
okeh

Reply:
on top of this i think u can add on a point at the end smth like "For all valid parameters and values, the field to be edited is updated successfully and shown in the status message" to generalise for all fields. but up to u tho if u find this useful